plugins/hls-semantic-tokens-plugin/src/Ide/Plugin/SemanticTokens/Internal.hs

```diff
@@ -60,23 +58,24 @@ import           Language.LSP.Protocol.Types          (NormalizedFilePath,
                                                        type (|?) (InL))
 import           Prelude                              hiding (span)
 
-logActionWith :: (MonadIO m) => IdeState -> Priority -> String -> m ()
-logActionWith st prior = liftIO . logPriority (ideLogger st) prior . T.pack
 
 -----------------------
 ---- the api
 -----------------------
 
-computeSemanticTokens :: IdeState -> NormalizedFilePath -> ExceptT PluginError Action SemanticTokens
-computeSemanticTokens st nfp = do
-  logActionWith st Debug $ "Computing semantic tokens:" <> show nfp
+computeSemanticTokens :: Recorder (WithPriority SemanticLog) -> PluginId -> IdeState -> NormalizedFilePath -> ExceptT PluginError Action SemanticTokens
+computeSemanticTokens recorder pid _ nfp = do
+  logWith recorder Debug (LogMsg "computeSemanticTokens start")
```

Comment:
I think this is probably not useful usually. Or we should be logging such things generically...

Reply:
removed